### PR TITLE
Custom help message, dont send admins to submitted.html, add vote!status to check ur entry

### PR DESCRIPTION
**Pull request by MrMyastan**
Feel free to change the help message or add extra functionality to it, I didn't know what we wanted so I just used the help message func
Also, I don't think the pm_help option does anything (I couldn't find it in the docs) but I don't want to break anything so I didn't touch it

### CODE REVIEW

**Comment by MrMyastan** [comment]
#26

**Comment by MrMyastan** [comment]
resolve #17 and #18

for 17:
In file_post_handler() store whether the auth key is a valid admin key in a var and then before it was returning 200 with the submitted page check that var and if its an admin key return a 303 redirect to the admin panel with the authkey (would've liked to use 205 for simplicity but that was causing it to try to download a file with the name of the admin key sooooo)

for 18:
Created vote!status that gives the message admins get about entries but for your own entry (only in dms)
I extracted the message generation out of submission_message() into entry_info_message() so we don't have that code twice
you could change it to a different message but we already had that code lying around and it has all the info sooooo

File: bot.py [hunk 1]
```diff
@@ -414,6 +418,22 @@ async def howmany(context: commands.Context) -> None:
...
+async def status(context: commands.Context) -> None:
+    
+    week = compo.get_week(True)
+
+    for entry in week["entries"]:
+        if entry["discordID"] == context.author.id:
+            await context.send(entry_info_message(entry))
+            return
+    
+    await context.send(f"You haven't submitted anything yet! But if you want to you can with %ssubmit !" % client.command_prefix[0])
```
Comment (wilm0x42) [inline bot.py:436]: Other than this, looks good to merge
```suggestion
    await context.send("You haven't submitted anything yet! But if you want to you can with %ssubmit !" % client.command_prefix[0])
```

**Comment by wilm0x42** [comment]
Plz make feature branches, so we have one feature per PR-- will make this speedier next time ;)

**Comment by MrMyastan** [comment]
Sorry, didnt know it would push right into the pr, will do next time

**Pull request merged**